packages/core/admin/admin/src/components/GuidedTour/Tours.tsx

```diff
@@ -291,30 +310,36 @@ const tours = {
           />
           <Step.Content
             id="tours.apiTokens.CopyAPIToken.content"
```

Comment:
When I click on the copy button, I get 2 notifications:
- Token copied to clipboard (expected)
- API Token successfully edited (unexpected)

I think the second one might be because the copy button triggers a form submission